test-one/src/index.js

```diff
@@ -1,14 +1,33 @@
 import { log } from "./helpers/logger";
 
-export const go = async () => {
-  // Your code should go here instead of the welcome message below. Create and modify additional files as necessary/appropriate.
+const isMultiple = (val, by) => {
+  return (val % by === 0)
+}
 
-  const welcomeMessage =
-    "Congrats, you're up and running! Welcome to the first moshtix code challenge! " +
-    "Please replace this message with your solution and feel free to add " +
-    "or modify other files as necessary. Good luck : )";
+export const BossHog = (val) => {
+  switch (true) {
+    case isMultiple(val, 15): 
```

Comment:
> Really great submission, did you get inspiration for your solution to test one anywhere, or is this all your own original ideas?

When working through any problem I consider 3 main things:
1. Have I done this before. 
2. How to make it modular. 
3. What's the naive solution and then what is a better solution. 

For test one, I answered the above as follows:
1. This test is similar to FizzBuzz coding challenges. I have seen these around since I began coding in 2015 so I am familiar with how to answer it. 
2. When learning how to write tests, the major lesson I learnt was to test the functionality of a function not the data. So I aimed to be able to check the function without being forced to check the loop data. 
3. The naive solution would be a for loop from 1 to 100 with a few if and else if statements. When considering the tests and the best way to write them, I did some research on the best way to test loops with jest and quickly realised I didn't need to check all of the data (as the lesson above says). This lead to the way I wrote it, so I could test just the BossHog function.